tools/hsettp/hsettp.c

```diff
@@ -1661,6 +1655,14 @@ main(const int argc, char **const argv)
     if (rc || head_to_exit)
         goto out;
 
+    if (optind >= argc) {
+        syntax("Socket path/KVDB home required");
+        rc = EX_USAGE;
+        goto out;
+    }
```

Comment:
I would lead with "KVDB home" as I suspect socket path is going to be rarely used/needed and will just confuse people when they try to figure out what it means and where it is...  Same in the usage message...

Reply:
Cool. Sounds good to me